openreview/venue/process/pc_submission_revision_process.py

```diff
@@ -38,4 +38,19 @@ def process(client, edit, invitation):
                     id = author_group.id,
                     members = submission_authors
                 )
+            )
+
+    invitation_invitations = [i for i in client.get_all_invitations(prefix=f'{venue_id}/-/', type='invitation') if i.is_active()]
+
+    for venue_invitation in invitation_invitations:
+        print('processing invitation: ', venue_invitation.id)
+        authorids = venue_invitation.edit['invitation'].get('edit', {}).get('note', {}).get('content', {}).get('authorids', {}).get('value', [])
+        if '${{4/id}/content/authorids/value}' in authorids:
```

Comment:
This should always be the path, correct? Or should I just check that `content/authorids/value` is in authorids[0]?

Reply:
I guess this should always be the path